src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ CHANGELOG
 
  * Added XSD for configuration
  * Added security configuration for priority-based access decision strategy
+ * Marked the `AbstractFactory`, `AnonymousFactory`, `FormLoginFactory`, `FormLoginLdapFactory`, `GuardAuthenticationFactory`, `HttpBasicFactory`, `HttpBasicLdapFactory`, `JsonLoginFactory`, `JsonLoginLdapFactory`, `RememberMeFactory`, `RemoteUserFactory` and `X509Factory` as `@final`
```

Comment:
as `@internal` :)

Reply:
Doh, fixed + also added this line to the UPGRADE-5.1 guide.